filestore/filestore.go

```diff
@@ -62,5 +64,17 @@ func (fs fileStore) Store(p Path, src File) error {
 }
 
 func (fs fileStore) Delete(p Path) error {
-	return os.Remove(fs.filename(p))
+	err := os.Remove(string(p))
```

Comment:
Seems like this could be changed to always delete something after constructing a path to within the filestore.  Safer also,  as there could be a relative path collision in and out of the filestore?